docs/content/manual/manual.yml

```diff
@@ -906,15 +906,42 @@ sections:
       - title: "`map(f)`, `map_values(f)`"
         body: |
 
-          For any filter `f`, `map(f)` will run that filter for each
-          element of the input array, and return the outputs in a new
-          array. `map(.+1)` will increment each element of an array of numbers.
+          For any filter `f`, `map(f)` and `map_values(f)` apply `f`
+          to each of the values in the input array or object, that is,
+          to the values of `.[]`.
+          
+          In the absence of errors, `map(f)` always outputs an array
+          whereas `map_values(f)` outputs an array or an object
+          depending on the type of the input.
```

Comment:
`In the absence of errors, ` is not needed.

Can you add something about the objects' keys being preserved?  Something like:

```
          `map(f)` always outputs an array whereas `map_values(f)` outputs
          an array if given an array, or an object if given an object.  In the
          case of `map_values(f)` applied to an object, the object output
          has all the same keys as the input object minus any for whose
          values `f` produced no new values.
```

something like that.

Reply:
Done. Tx.